shared/pubsub.js

```diff
@@ -203,15 +203,6 @@ const defaultClientEventHandlers = {
     client.socket = null
     client.clearAllTimers()
 
-    // See "Status Codes" https://tools.ietf.org/html/rfc6455#section-7.4
-    switch (event.code) {
-      // TODO: verify that this list of codes is correct.
-      case 1000: case 1002: case 1003: case 1007: case 1008: {
-        client.shouldReconnect = false
-        break
-      }
-      default: break
-    }
```

Comment:
Instead of outright deleting this code, as the RFC comment is useful, could you just comment it out and add another comment with a link to https://github.com/okTurtles/group-income/issues/1246 ?